Makefile

```diff
@@ -262,41 +262,49 @@ bootstrap-minikube:
 	hack/minikube-cluster.sh
 	$(MAKE) set-context-minikube cluster-bootstrap
 
-## -- gke
+## -- clouds
 
-require-gcloud-project:
-ifndef GCLOUD_PROJECT
-	$(error GCLOUD_PROJECT not set)
-endif
-
-DEPLOYER=./hack/deployer/deployer --plans-file=hack/deployer/config/plans.yml --config-file=hack/deployer/config/deployer-config.yml
+PROVIDER=$(shell test -f hack/deployer/config/provider && cat hack/deployer/config/provider)
+DEPLOYER=./hack/deployer/deployer --plans-file=hack/deployer/config/plans.yml --config-file=hack/deployer/config/deployer-config-$(PROVIDER).yml
 
 build-deployer:
 	@ go build -mod=readonly -o ./hack/deployer/deployer ./hack/deployer/main.go
 
-setup-deployer-for-gke-once: require-gcloud-project build-deployer
-ifeq (,$(wildcard hack/deployer/config/deployer-config.yml))
-	@ ./hack/deployer/deployer create defaultConfig --path=hack/deployer/config/deployer-config.yml
+create-default-config:
+ifeq (,$(wildcard hack/deployer/config/deployer-config-$(PROVIDER).yml))
```

Comment:
Nit? we seem to be a bit inconsistent with the order of `ifeq` args here and elsewhere